nix/sources.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This whole nix dir should be nuked IMO

Reply:
this is auto-generated from niv, would you like to keep or remove from the default.nix?

(I had tried deleting and commiting but it looks like it showed after pushing rebase changes)

Reply:
Please nuke and if still green that is better. I don’t think it was auto generated, I think I added it.

Reply:
@harlantwood @dauphin3 

When I spoke with Zeek, he said he did nuke it, and then they popped back up. Zeek can you try that again to confirm your actions?

Reply:
got it this time, but checks not passing